subsys/bluetooth/controller/ll_sw/pdu.h

```diff
@@ -285,6 +285,8 @@ enum pdu_adv_mode {
 	EXT_ADV_MODE_NON_CONN_SCAN = 0x02,
 };
 
+#define PDU_ADV_SID_COUNT 16
```

Comment:
We also have the `BT_GAP_SID_MAX` define; not sure if it makes sense to use that in the controller.

Reply:
I did notice that, which was not a count but the maximum value, also it is in the GAP namespace.